src/pricing/index.njk

```diff
@@ -167,20 +148,16 @@ hubspot:
                     <!-- Enterprise -->
                     {% macro enterprise(show_sla=true, cloud=true) %}
                     <div class="pricing-tile md:h-full">
-                        <div class="pricing-tile-background pb-6 md:grid flex-col">
+                        <div class="pricing-tile-background pb-6 md:grid md:grid-rows-[auto_1fr_minmax(36px,_36px)]">
                             <div>
-                                <div
-                                    class="flex flex-col items-center text-left md:flex-row md:mb-6 md:px-0 md:block md:min-h-[133px]">
+                                <div class="flex flex-col items-center text-left md:flex-row md:mb-6 md:px-0 md:block md:min-h-[133px]">
                                     <div class="flex flex-row justify-between mt-4 w-full gap-x-2">
                                         <h4 style="display:inline-block;">Enterprise</h4>
                                     </div>
-                                    <p class="mt-7">For <span class="inline font-normal">business critical
-                                            applications</span> that require high availability and enterprise support.
-                                    </p>
+                                    <p class="mt-7">For <span class="inline font-normal">business critical applications</span> that require high availability and enterprise support.</p>
                                 </div>
                                 <div class="flex flex-col content-start features md:mt-3 mt-6 w-full">
-                                    <note class="font-bold" style="display: inline-block;">Top Features:
-                                    </note>
+                                    <note class="font-bold" style="display: inline-block;">Top Features:</note>
```

Comment:
Why not do the "includes" here?

Reply:
@Yndira-E Can you create a second PR to align the enterprise tier? I'll merge this one now.